src/components/Contact/contact.css

```diff
@@ -1,192 +1,225 @@
-}
-
-.facebook-icon {
-    width: 4%;
-    top: 67.5%;
-    left: 71.25%;
-}
-
-.contact-container h1 {
-    position: absolute;
-    top: 0%;
-    left: 33%;
-    color: #ffffff;
-    font-size: 3.5vw;
-}
-
-.contact-hashtag {
-    position: absolute;
-    top: 70%;
-    left: 18%;
-    width: 30%;
-    color: #ffffff;
-    font-size: 1.75vw;
-}
-
-.contact-email {
-    position: absolute;
-    top: 40%;
-    left: 60%;
-    width: 25%;
-    color: #ffffff;
-    font-size: 1.75vw;
-}
-
-.contact-container a:hover {
-    transition: all 0.4s ease-in-out;
-    transform: scale(1.2);
-}
-
-@media screen and (max-width: 976px) {
-    .bg-lg {
-        display: none;
-    }
-    .bg-md {
-        display: block;
-    }
-    .reddit-icon {
-        width: 4.5%;
-        top: 46.5%;
-        left: 33.25%;
-    }
-    .twitter-icon {
-        width: 4.5%;
-        top: 31.75%;
-        left: 54%;
-    }
-    .instagram-icon {
-        width: 4.5%;
-        top: 74.25%;
-        left: 60.85%;
-    }
-    .facebook-icon {
-        width: 5%;
-        top: 68.25%;
-        left: 79.75%;
-    }
-    .contact-container h1 {
-        position: absolute;
-        top: -5%;
-        left: 29%;
-        color: #ffffff;
-        font-size: 4.5vw;
-    }
-    .contact-hashtag {
-        position: absolute;
-        top: 65%;
-        left: 10%;
-        width: 35%;
-        color: #ffffff;
-        font-size: 2.5vw;
-    }
-    .contact-email {
-        position: absolute;
-        top: 40%;
-        left: 65%;
-        width: 30%;
-        color: #ffffff;
-        font-size: 2.25vw;
-    }
-}
-
-@media screen and (max-width: 500px) {
-    .bg-md {
-        display: none;
-    }
-    .bg-sm {
-        display: block;
-    } 
-    .reddit-icon {
-        width: 8%;
-        top: 46%;
-        left: 13.5%;
-    }
-    .twitter-icon {
-        width: 7.5%;
-        top: 31.5%;
-        left: 45%;
-    }
-    .instagram-icon {
-        width: 7%;
-        top: 74%;
-        left: 55.25%;
-    }
-    .facebook-icon {
-        width: 8.5%;
-        top: 67.5%;
-        left: 83.5%;
-    }
-    .contact-container h1 {
-        position: absolute;
-        top: -5%;
-        left: 27.5%;
-        color: #ffffff;
-        font-size: 5vw;
-    }
-    .contact-hashtag {
-        position: absolute;
-        top: 70%;
-        left: 10%;
-        width: 34%;
-        color: #ffffff;
-        font-size: 3vw;
-    }
-    .contact-email {
-        position: absolute;
-        top: 40%;
-        left: 55%;
-        width: 34%;
-        color: #ffffff;
-        font-size: 3vw;
-    }
+.contact-container {
+    position: relative;
+    top: 0;
+    left: 0;
+}
+
+.bg-lg,
+.bg-md,
+.bg-sm,
+.social-icon {
+    width: 100%;
+    position: relative;
+    top: 0;
+    left: 0;
+}
+
+.bg-lg {
+    display: block;
+}
+
+.bg-md {
+    display: none;
+}
+
+.bg-sm {
+    display: none;
+}
+
+.reddit-icon,
+.twitter-icon,
+.instagram-icon,
+.facebook-icon,
+.linkedin-icon {

```

Comment:
Create and add a class to the html so the css can be changed to have just one class instead of all the classes.